text/0008-bounded-contexts.md

```diff
@@ -0,0 +1,25 @@
+* Start Date: 11/13/2023
+* Contributors:
+  * [Craig McNally](cmcnally@ebsco.com)
+  * [Vince Bareau](vbareau@ebsco.com)
+* RFC PRs:
+  * PRELIMINARY REVIEW: https://github.com/folio-org/rfcs/pull/20
+  * DRAFT REFINEMENT: TBD
+  * PUBLIC REVIEW: TBD
+  * FINAL REVIEW: TBD
+* Outcome: 
+
+# Bounded contexts
+
+## Summary
+It is proposed that the boundary of the microservices, and thus the bounded context, be extended beyond the module. The new bounded context would be found at the Application level. Thus, any of the backend modules in the Application would be able to directly access the storage layer.
```

Comment:
It would be useful to define what folks mean by `bounded context` and `microservice` in this context

Reply:
I didn't explicitly define these terms, but did reword the summary.  Let me know if it's more clear now, or if explicitly defining these terms is still necessary.

Reply:
Thank you, I think including an inline description in the summary is an improvement

Personally, I would still like the RFC to include definitions for these terms. I don't think that is urgent. It may just be me that has a fairly different understanding of those terms than their usage in this RFC

Reply:
OK, thanks @marcjohnson-kint there is still time to define these terms in the RFC.  For now I will resolve this thread and we can discuss it more in later stages of the RFC.